velox/row/tests/UnsafeRowFuzzTest.cpp

```diff
@@ -106,6 +106,7 @@ TEST_F(UnsafeRowFuzzTests, fast) {
       DOUBLE(),
       VARCHAR(),
       VARBINARY(),
+      UNKNOWN(),
```

Comment:
Do we support UNKNOWN in Arrays, Nested arrays and Map as well? Thanks!

Reply:
No, right now we only support the first level element to be UNKNOWN in ROW.